src/skmultiflow/trees/nodes/sst_inactive_learning_node.py

```diff
@@ -81,10 +81,10 @@ def update_weights(self, X, y, learning_ratio, rht):
         _, n_features = get_dimensions(X)
         _, n_targets = get_dimensions(y)
 
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```

Comment:
The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.